File: Python/成绩划分等级.py
import sys
x=int(sys.argv[1])
with open('result.txt','w') as fp:
	if 95<=x and x<=100 :
		fp.write('A')
	elif 85<=x and x<95 :
		fp.write('B')
	elif 70<=x and x<85 :
		fp.write('C')
	elif 60<=x and x<70 :
		fp.write('D')
	elif 0<=x and x<60 :
		fp.write('E')
	else :
		fp.write('I')

File: C/成绩划分等级.c
#include<stdio.h>
#include<stdlib.h>
int main(int argc,char** argv)
{
	FILE *fp;
	if((fp=fopen("result.txt","w"))==NULL)
		return -1;
	int x;
	x=atoi(argv[1]);
	if(95<=x&&x<=100) fprintf(fp,"A");
	else if(85<=x&&x<95) fprintf(fp,"B");
	else if(70<=x&&x<85) fprintf(fp,"C");
	else if(60<=x&&x<70) fprintf(fp,"D");
	else if(0<=x&&x<60) fprintf(fp,"E");
	else fprintf(fp,"I"); 
	fclose(fp);
	return 0;
 } 
